test/ClangImporter/objc_override.swift

```diff
@@ -116,6 +116,16 @@ class MyHashableNSObject: NSObject {
   }
 }
 
+// rdar://problem/47557376
+// Adding an override to someone else's class in an extension like this isn't
+// really sound, but it's allowed in Objective-C too.
+extension OverrideInExtensionSub {
+  open override func method() {}
+}
+public extension OverrideInExtensionSub {
+  open override func accessWarning() {} // expected-warning {{declaring open instance method in a public extension}}
```

Comment:
I know this diagnostic already exists, but "declaring open instance method in a public extension" isn't great. It doesn't really give you any idea of what the compiler will do with this code or how you should correct the issue. Perhaps the fix-it normally provides enough additional information to understand it, but there's no fix-it here. I think it should say something like "public keyword on extension will be ignored for this open instance method".